packages/botonic-react/src/components/custom-message.jsx

```diff
@@ -77,7 +79,14 @@ export const customMessage = ({
   WrappedComponent.customTypeName = name
   // eslint-disable-next-line react/display-name
   WrappedComponent.deserialize = msg => (
-    <WrappedComponent id={msg.id} key={msg.key} json={msg.data} {...msg.data} />
+    <WrappedComponent
+      id={msg.id}
+      key={msg.key}
+      json={msg.data}
```

Comment:
Observation: I know this prop was alredy passed into this component, but taking a look at the `WrappedComponent`, is the `json` prop really needed here?

Reply:
Yes it is strange how this is passed here and it is recreated inside the CustomMessage. In a log you can see how one of the props is json and inside there is the same json with less attributes. 
![Captura de pantalla 2023-12-14 a las 15 18 53](https://github.com/hubtype/botonic/assets/36898236/f430488d-3f3a-4988-99d5-f0d819b987ef)

For now I don't want to touch it but I will check it in the future in another PR.